Symfony/CS/Tests/Fixtures/Integration/empty_return,useless_return,whitespacy_lines,extra_empty_lines.test

```diff
@@ -0,0 +1,43 @@
+--TEST--
+Integration of fixers: empty_return,useless_return,whitespacy_lines,extra_empty_lines.
+--CONFIG--
+level=none
+fixers=empty_return,useless_return,whitespacy_lines,extra_empty_lines
+--INPUT--
+<?php
+class Test {
+
+    private static function a($a) {
+        if ($a) {
+            return 1;
+        }
+        return ;
+    }
+
+    private static function b($a) {
+        if ($a) {
+            return;
+        }
+
+        return null;
+    }
+}
+
+--EXPECT--
+<?php
+class Test {
+
+    private static function a($a) {
+        if ($a) {
+            return 1;
+        }
+
```

Comment:
The `extra_empty_lines` should pick this one up, we should add more config options to that one.
Which reminds me, this fixer should also run before the `return` fixer to prevent that one from adding an empty line.